packages/IMAPClient-UI.package/ICLoginDialog.class/instance/changeInputField..st

```diff
@@ -0,0 +1,24 @@
+helper
+changeInputField: currentIndex
+
+	| correctSubmorph text |
+
+	"last element has no successor"
+	(self inputCollection size > (currentIndex))
+	ifTrue: [
+		"find submorph with same symbol as successor "
+		correctSubmorph := self inputCollection detect: [:each | 
+				(each getTextSelector = (self symbolArray at: (currentIndex + 1)))
+			]. 	
```

Comment:
I personally feel like we shouldn't use brackets like that, because I feel like we got taught in SWA that this isn't the best way to align the code. BUT on the other hand I searched for examples how to align code properly but even in the Morph class there are a LOT of inconsistencies. So I think we should propably sit down once as a team and establish some rules.

Reply:
So there are a lot of discussions about that. See here: http://live.exept.de/doc/online/english/programming/codingStyle.html#CODE_INDENTATION. I feel most comfortable with doing C-indentation style used by Kernigham & Ritchie. 

C Style Indendation is like: 

(example borrowed from document above)
```
foo
	"this method performs some fooBar.
	 Sometimes even baz is done"

	doingBar ifTrue:[
	    self fooBar.
	    [doingBaz] whileTrue:[
		self baz
	    ].
	    self moreFooBar.
	    1 to:10 do:[:index |
		1 to:10 do:[:index2 |
		    self doMore
		]
	    ]
	] ifFalse:[
	    ...
	    and so on
	]
```
The other way of doing it is Smalltalk 80/LISP style: 
``` 
    foo
	"this method performs some fooBar.
	 Sometimes even baz is done"

	doingBar
	    ifTrue:[
		self fooBar.
		[doingBaz]
		    whileTrue:[
			self baz].
		self moreFooBar.
		1 to:10 do:[:index |
		    1 to:10 do:[:index2 |
			self doMore]]]
	    ifFalse:[
		...
		and so on]
``` 
I think its important to decide for one style. So i would vote for the C-Style. Its easier to read, and taken from our daily coding live in other languages.